.gitlab-ci.yml

```
@@ -332,6 +332,20 @@ publish-benchmarks-s3:
   after_script:
     - aws s3 ls s3://${BUCKET}/${PREFIX}/ --recursive --human-readable --summarize
 
+# Ref: https://github.com/paritytech/opstooling/issues/111
+update-parachain-template:
+  stage:                           publish
+  extends:                         .docker-env
```

```suggestion
  extends:                         .kubernetes-env
```
It seems that the job is quite fast, we can execute it in k8s runner